Extensions/Emails/Services/EmailSenderShellScopeExtensions.cs

```diff
@@ -8,13 +8,17 @@
 
 namespace Lombiq.HelpfulExtensions.Extensions.Emails.Services
 {
-    public class EmailService : IEmailService
+    public static class EmailSenderShellScopeExtensions

```

Comment:
Since it's now Extensions, it shouldn't remain in the Services namespace.

Reply:
This wasn't addressed.

Reply:
Finally, it's Lombiq.Helpful**Extensions**.**Extensions**.Emails.**Extensions**!
I guess this is what you call an extended namespace? 🙃